components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/SAMLSSOConstants.java

```diff
@@ -87,6 +87,15 @@
     public static final String SAML_SSO_SP_REQUEST_VALIDATOR_CONFIG_PATH = "SSOService.SAMLSSOSPRequestValidator";
     public static final String INBOUND_AUTH_TYPE_SAML = "samlsso";
 
+    // SAML2 Artifact Binding
+    public static final byte[] SAML2_ARTIFACT_TYPE_CODE = { 0, 4 };
+    public static final String SAML_ART = "SAMLart";
+    public static final String SAML_ARTIFACT_RESOLVE_URL = "/samlartresolve";
```

Comment:
Is samlartresolve value defined from the spec? if not can we define more clear value like /samlartifactresolve?

Reply:
No this is not specified in the spec. We should discuss on a more suitable endpoint name. I will send a mail regarding this.

Reply:
What was the resolution on this?